homeassistant/components/simplefin/strings.json

```diff
@@ -6,6 +6,13 @@
         "data": {
           "api_token": "Claim Token or Access URL"
         }
+      },
+      "reauth_confirm": {
+        "title": "SimpleFin - Re-authentication Required",
+        "description": "Please enter either a new Claim Token or an Access URL.",
+        "data": {
+          "api_token": "Claim Token or Access URL"
```

Comment:
Where is this reference in the reauth flow? the field i see has a different name (`access_url`)